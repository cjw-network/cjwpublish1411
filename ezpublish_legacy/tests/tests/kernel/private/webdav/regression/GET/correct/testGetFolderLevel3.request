<?php

// Test if GET returns correct information on the folder $GLOBALS['ezc_webdav_testfolder']

$_SERVER['REQUEST_URI'] = '/' . $GLOBALS['ezc_siteaccess'] . '/Content/' . $GLOBALS['ezc_webdav_testfolder'] . '/';
$_SERVER['REQUEST_METHOD'] = 'GET';

?>
