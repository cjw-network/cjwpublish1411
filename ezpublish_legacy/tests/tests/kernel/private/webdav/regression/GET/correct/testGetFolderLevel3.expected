Content-Length: 4096
ETag: XXX
Server: eZComponents/dev/ezcWebdavTransportWrapper
HTTP/1.1 200 OK
